core/src/wallet/common/synchronizers/AbstractBlockchainExplorerAccountSynchronizer.h

```diff
@@ -246,6 +246,8 @@ namespace ledger {
                         buddy->token = Option<void *>(t);
                     }
                     return unit;
+                }).template flatMap<Unit>(account->getContext(), [buddy, self](const Unit&) {
+                    return self->extendKeychain(0, buddy);
```

Comment:
We can do a small optim by starting to extend the keychain starting by the last known batch.

If we already synchronize the wallet once and know that there is 2 batches. Next time we only need to try to fetch batch 3. If batch 3 has some. transaction we can continue. to fetch more batches, if it doesn't we can stop "extendKeychain" after one http call instead of 2 with the current code.